server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -268,13 +271,13 @@ module.exports = {
       visible: validAddress,
     }
 
-    const informOffenderTask = {
+    const invisibleInformOffenderTask = {
       task: 'informOffenderTask',
       visible: true,
```

Comment:
If it's invisible why is`visible:true`?

Reply:
I think its a bug in the code.  I created an informOffenderTask at the module level, then discovered this one at the function level so i renamed it.  I don't think it is ever rendered and I suppose I should investigate and remove it.  The Eligibility task post approval and the way it works looks rather broken, but I can’t be be sure of that, no one has complained and  it wasn’t part of the ticket so I ignored it.